easybuild/easyconfigs/e/ecFlow/ecFlow-5.7.0-GCC-10.2.0.eb

```
@@ -0,0 +1,45 @@
+# Written by Tom Strempel <tom.strempel@ufz.de>
+# Modified by Ben Langenberg <ben.langenberg@ufz.de>
+easyblock = "CMakeMake"
+
+name = 'ecFlow'
+version = '5.7.0'
+
+homepage = "https://confluence.ecmwf.int/display/ECFLOW/ecflow+home"
+description = """ecFlow is a client/server workflow package that enables users
+                 to run a large number of programs (with dependencies on each other and on time)
+                 in a controlled environment. It provides reasonable tolerance for hardware and
+                 software failures, combined with restart capabilities.
+                 It is used at ECMWF to run all our operational suites across a range of platforms."""
```

nitpick: the whitespace will be the same in the generated module file; I'd recommend something like this:

```suggestion
description = """ecFlow is a client/server workflow package that enables users
 to run a large number of programs (with dependencies on each other and on time)
 in a controlled environment. It provides reasonable tolerance for hardware and
 software failures, combined with restart capabilities.
 It is used at ECMWF to run all our operational suites across a range of platforms."""
```